velox/exec/ExchangeSource.h

```diff
@@ -94,7 +95,7 @@ class ExchangeSource : public std::enable_shared_from_this<ExchangeSource> {
   // Returns runtime statistics. ExchangeSource is expected to report
   // background CPU time by including a runtime metric named
   // ExchangeClient::kBackgroundCpuTimeMs.
-  virtual folly::F14FastMap<std::string, int64_t> stats() const = 0;
+  virtual folly::F14FastMap<std::string, RuntimeCounter> stats() const = 0;
```

Comment:
Thanks for the improvement, I also encountered the same problem.

I propose to go one step further: **use `RuntimeMetric` directly**. I noticed that currently `RuntimeCounter` is only used to help update a `RuntimeMetric`, there is no place to use it directly.

Using `RuntimeMetric` also allows us to see the min/max information of an indicator, so that we no longer need to explicitly define some maxXxx to record these indicators.
For example, if we want to add an metric in a subclass (perhaps `PrestoExchangeSource`) to record the time spent on network flying (total time spent & maximum RT), if we use `int64_t`/`RuntimeCounter`, we need to define two variables(eg, flyingTime, maxFlyingTime). But if we use `RuntimeMetric`, it will automatically help us record the min/max values, so we only need to define one (eg, `RuntimeMetric flyingTime`).

@mbasmanova  What do you think?

Reply:
1、ExchangeClient summarizes all current indicators for all ExchangeSources.
2、ExchangeClient provides **RuntimeMetric** to Exchange.

Reply:
@lingbin I like this idea.

@heary-cloud Perhaps, we can use RuntimeMetric::merge to combine these.